README.md

```diff
@@ -1 +1,41 @@
-# js-capstone
\ No newline at end of file
+# mobile-portfolio
+This  project is aimed at building a beautiful porfolio page.
+
+# Built With
+HTML
+CSS
+JAVASCRIPT
+
+# Video presentation
+
+[MOVIES demo](https://youtu.be/PdEa4KJuPNk)
+
+
+
+
+# live Demo
+ To see a live demo kindly visit [MOVIES](https://amklass11.github.io/Mobile-portfolio/)
+
+# Getting Started
+This is a  microverse project aimed at grasping JAVASCRIPTS wtih html
+ css and js by building the Movie app of the website. 
```

Comment:
- [ ] The getting started section of the readme file should describe how we setup and run the project on local environment, kindly put the description at the top in description section